Logger/FileLogger.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private readonly string FilePath;
+
+        public FileLogger(string filePath)
+        {
+            FilePath = filePath;
+        }
+
+        public override void Log(LogLevel log, string message)
+        {
+            using (StreamWriter streamWriter = File.AppendText(FilePath))
+            {
+                streamWriter.WriteLine(DateTime.Now.ToString("MM/dd/yyyy hh:mm:ss tt") + ' ' + ClassName + ' ' + Enum.GetName(typeof(LogLevel), log) + ": " + message);
+            }
```

Comment:
This works, but there is a new C# syntax that can reduce nesting. This will dispose of the object when it leaves scope.
It is also typically cleaner to use [string interpolation](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated) rather than concatenating string. You can also simply call ToString on an enum member to get its name.
```suggestion
            using StreamWriter streamWriter = File.AppendText(FilePath);
            streamWriter.WriteLine($"{DateTime.Now:"MM/dd/yyyy hh:mm:ss tt"} {ClassName} {log}: {message}");
```